Libraries/Image/Image.ios.js

```diff
@@ -53,7 +53,7 @@ const ImageViewManager = NativeModules.ImageViewManager;
  *         />
  *         <Image
  *           style={{width: 50, height: 50}}
- *           source={{uri: 'https://facebook.github.io/react/logo-og.png'}}
+ *           source={{uri: 'https://facebook.github.io/react-native/img/logo-og.png'}}
```

Comment:
This image is not loading for me. The opengraph image for RN is https://facebook.github.io/react-native/img/opengraph.png

Reply:
I prefer `logo-og.png` since the logo is in the center and has a dark background and all the examples in RNTester fit better, what do you think? 

![image](https://user-images.githubusercontent.com/2577356/31355855-4054be9a-ad12-11e7-88ef-2f3b2c948329.png)

Reply:
Sounds good to me.